### PR TITLE
purging repos

### PR DESCRIPTION
**Pull request by jiribroulik**

### CODE REVIEW

**Comment by cloudlab-autobuild** [comment]
Can one of the admins verify this patch?

File: linux/system/repo.sls [hunk 1]
```diff
@@ -28,6 +28,15 @@ include:
 
 {% set default_repos = {} %}
 
+{%- if system.purge_repos %}
+
+remove_sources_list_d_repos:
+   file.directory:
+      - name: /etc/apt/sources.list.d/
```
Comment (fpytloun) [inline linux/system/repo.sls]: Too many spaces (use 2 spaces per indent)

File: linux/system/repo.sls [hunk 2]
```diff
@@ -28,6 +28,15 @@ include:
 
 {% set default_repos = {} %}
 
+{%- if system.purge_repos %}
```
Comment (fpytloun) [inline linux/system/repo.sls]: if system.purge_repos|default(False)

File: linux/system/repo.sls [hunk 3]
```diff
@@ -28,6 +28,15 @@ include:
...
+
+remove_sources_list_d_repos:
```
Comment (fpytloun) [inline linux/system/repo.sls]: purge_sources_list_d

**Pull request merged**